openmc/deplete/abc.py

```diff
@@ -918,6 +918,24 @@ def add_transfer_rate(
         self.transfer_rates.set_transfer_rate(material, components, transfer_rate,
                                       transfer_rate_units, destination_material)
 
+    def add_redox(self, material, buffer, oxidation_states):
+        """Add redox control to depletable material.
+
+        Parameters
+        ----------
+        material : openmc.Material or str or int
+            Depletable material
+        buffer : dict
+            Dictionary of buffer nuclides to be added to keep redox constant,
+            where keys are nuclide names and values fractions to 1.
```

Comment:
Is there a use case where the value would be something other than 1? It's not entirely clear to me what the meaning of the value is in this dictionary